app/src/main/res/layout/fragment_comments_reply.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="?attr/windowBackground">
+
+    <androidx.appcompat.widget.Toolbar
+        android:id="@+id/toolbar"
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        android:layout_alignParentTop="true"
+        android:gravity="center_vertical"
+        android:minHeight="?attr/actionBarSize">
+
+        <ImageButton
+            android:id="@+id/backButton"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_margin="@dimen/activity_horizontal_margin"
+            android:background="?attr/selectableItemBackgroundBorderless"
+            android:contentDescription="@string/back"
+            android:src="@drawable/ic_arrow_back" />
```

Comment:
I think you can just use [android:navigationIcon](https://developer.android.com/reference/android/widget/Toolbar#attr_android:navigationIcon) and [android:navigationContentDescription](https://developer.android.com/reference/android/widget/Toolbar#attr_android:navigationContentDescription) instead of adding a button manually

Reply:
I can not use your way to do that, it will show nothing at here.
There is [another way](https://developer.android.com/develop/ui/views/components/appbar/actions) can do that, but it's even more complex.
![图片](https://user-images.githubusercontent.com/32761048/206725614-f769cc74-ec43-4fab-9d9c-efea6a87da47.png)